lib/tasks/enrichManifest.js

```diff
@@ -0,0 +1,31 @@
+import manifestEnricher from "../processors/manifestEnricher.js";
+import fsInterface from "@ui5/fs/fsInterface";
+
+/* eslint "jsdoc/check-param-names": ["error", {"disableExtraPropertyReporting":true}] */
+/**
+ * Task for transforming the manifest.json file.
+ *
+ * @public
+ * @function default
+ * @static
+ *
+ * @param {object} parameters Parameters
+ * @param {@ui5/fs/DuplexCollection} parameters.workspace DuplexCollection to read and write files
+ * @param {object} parameters.options Options
+ * @param {string} parameters.options.projectNamespace Namespace of the application
+ * @returns {Promise<undefined>} Promise resolving with <code>undefined</code> once data has been written
+ */
+export default async function({workspace, options}) {
+	const {projectNamespace} = options;
+
+	// Note: all "manifest.json" files in the given namespace
+	const resources = await workspace.byGlob(`/resources/${projectNamespace}/**/manifest.json`);
+
+	const processedResources = await manifestEnricher({
```

Comment:
After looking at this again and again I still stumble over the name manifest**Enricher**. Even though we agreed on this name a while ago, personally I think I would prefer a different name at this time.

### Definition of _enrichment_:
> the act or process of improving the quality or power of something by adding something else

This is indeed fitting, however the thing that does the enrichment is usually called "enrichment *something* (e.g. facility)" rather than "*something* enricher". See also [wiktionary](https://en.wiktionary.org/wiki/enricher)

### Definition for _augmentation_:

> the action or process of making or becoming greater in size or amount.

I find this fitting too and "[manifestAugmenter](https://en.wiktionary.org/wiki/augmenter)" works much better in my opinion.

### Definition for _enhancement_:

> an increase or improvement in quality, value, or extent.

"[manifestEnhancer](https://en.wiktionary.org/wiki/enhancer)"  also sounds better to me.


### Options

1. manifestEnricher (enrichManifest)
2. manifestAugmenter (augmentManifest)
3. manifestEnhancer (enhanceManifest)

Reply:
@KlattG any recommendations from your side. I am fine with all options mentioned by @RandomByte

Reply:
@flovogt I like enhancer best, augmenter second, and enricher would be my third choice.

Reply:
rename done